docs/helpers/env_vars.yaml

```diff
@@ -8075,6 +8280,15 @@ OCIS_GRPC_CLIENT_TLS_MODE:
   deprecationVersion: ""
   removalVersion: ""
   deprecationInfo: ""
+OCIS_GRPC_PROTOCOL:
+  name: OCIS_GRPC_PROTOCOL;STORAGE_PUBLICLINK_GRPC_PROTOCOL
+  defaultValue: ""
+  type: string
+  description: The transport protocol of the GRPC service.
+  introductionVersion: pre5.0
```

Comment:
How can this be introduced `pre5.0` but it is only coming here to the docs now?

Reply:
**Explanation:**
* The `auth-bearer` service was introduced in version 3, see the following [table](https://owncloud.dev/services/general-info/added-services/).
* Envvars have been created at that time like the `AUTH_BEARER_GRPC_PROTOCOL`.
* With v5, we decided to create in introduction version to see which envvars have been introduced with which version. The above envvar got `pre5.0` as it was introduced before 5.
* Now, with one of the recent code changes, the following happened:
The existing `AUTH_BEARER_GRPC_PROTOCOL` envvar (pre5.0) got a global envvar `OCIS_GRPC_PROTOCOL` **added**. This does not change the introduction version of the existing envvar as it is added only.

Your question is really good and highlights that we should write some dev docs notes about in [Lifecycle of Envvars](https://owncloud.dev/services/general-info/envvar-scopes/#lifecycle-of-envvars).